scripts/globals/items/tube_of_clear_salve_i.lua

```diff
@@ -0,0 +1,44 @@
+-----------------------------------------
+-- ID: 5837
+-- Item: tube_of_clear_salve_i
+-- Item Effect: Instantly removes one negative status effect from pet
+-----------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    if (not target:hasPet()) then
+        return tpz.msg.basic.REQUIRES_A_PET
+    end
+    return 0
+end
+
+function onItemUse(target)
+    local pet = target:getPet()
+    
+    local function removeStatus()
+        if pet:delStatusEffect(tpz.effect.PETRIFICATION) then return true end
+        if pet:delStatusEffect(tpz.effect.SILENCE) then return true end
+        if pet:delStatusEffect(tpz.effect.BANE) then return true end
+        if pet:delStatusEffect(tpz.effect.CURSE_II) then return true end
+        if pet:delStatusEffect(tpz.effect.CURSE) then return true end
+        if pet:delStatusEffect(tpz.effect.PARALYSIS) then return true end
+        if pet:delStatusEffect(tpz.effect.PLAGUE) then return true end
+        if pet:delStatusEffect(tpz.effect.POISON) then return true end
+        if pet:delStatusEffect(tpz.effect.DISEASE) then return true end
+        if pet:delStatusEffect(tpz.effect.BLINDNESS) then return true end
+        if pet:eraseStatusEffect() ~= 255 then return true end
+        return false
+    end
```

Comment:
I know this section probably exists somewhere else, but there _has_ to be a better way of doing this...